Tests/scripts/update_id_set.py

```diff
@@ -348,13 +358,66 @@ def process_testplaybook_path(file_path):
     print("adding {0} to id_set".format(file_path))
     script = None
     playbook = None
-    if re.match(TEST_SCRIPT_REGEX, file_path, re.IGNORECASE):
+    if checked_type(file_path, (TEST_SCRIPT_REGEX, PACKS_TEST_PLAYBOOKS_REGEX)):
         script = get_script_data(file_path)
-    elif re.match(TEST_PLAYBOOK_REGEX, file_path, re.IGNORECASE):
+    elif checked_type(file_path, (TEST_PLAYBOOK_REGEX, PACKS_TEST_PLAYBOOKS_REGEX)):
         playbook = get_playbook_data(file_path)
+
     return playbook, script
 
 
+def get_integrations_paths():
+    path_list = [
+        ['Integrations', '*'],
+        ['Beta_Integrations', '*'],
+        ['Packs', '*', 'Integrations', '*', '*.yml'],  # Case of package
```

Comment:
why are you considering packages only for packs and not integrations/beta?
no need to split, the `process_integration` will handle that